flink-table/flink-table-planner/src/test/java/org/apache/flink/table/runtime/batch/table/JavaTableEnvironmentITCase.java

```diff
@@ -156,17 +156,6 @@ public void testTableRegister() throws Exception {
 		compareResultAsText(results, expected);
 	}
 
-	@Test(expected = TableException.class)
-	public void testIllegalName() throws Exception {
```

Comment:
As mentioned earlier, let's keep this test but modify it with other illegal names.